DXRModules/DeusEx/Classes/DXREvents.uc

```diff
@@ -843,6 +857,24 @@ static function AddPawnDeath(ScriptedPawn victim, optional Actor Killer, optiona
         e._AddPawnDeath(victim, Killer, damageType, HitLocation);
 }
 
+function bool checkInitialAlliance(ScriptedPawn p,name allianceName, float allianceLevel)
+{
+    local int i;
+
+    for (i=0;i<8;i++){
+        if (p.InitialAlliances[i].AllianceName==allianceName &&
+            p.InitialAlliances[i].AllianceLevel==allianceLevel){
```

Comment:
should probably use ~= operator for this float comparison (equality within 0.0001), just in case